api4/schema.graphqls

```diff
@@ -85,24 +90,40 @@ type ChannelMember {
 type User {
 	id: String!
 	username: String!
+	# deliberately omitting password
 	email: String!
+	authData: String
 	firstName: String!
 	lastName: String!
 	nickname: String!
+	emailVerified: Boolean!
 	isBot: Boolean!
 	isGuest: Boolean!
 	isSystemAdmin: Boolean!
 	createAt: Float!
+	updateAt: Float!
 	deleteAt: Float!
 	authService: String!
 	customStatus: CustomStatus
 	status: Status
 	props: StringMap!
 	notifyProps: StringMap!
-	lastPictureUpdateAt: Float!
+	lastPictureUpdate: Float!
+	lastPasswordUpdate: Float!
```

Comment:
This isn't specific to the new changes, but a few of these other fields are also being conditionally sanitized out in websocket messages and REST API responses. Do we have to do anything in the resolver to prevent users from accessing fields they shouldn't have access to?

Reply:
When they are sanitized, they will automatically become zero. Nothing additional needs to be done from resolver side.